### DISCUSSION TITLE
Anybody a hypridle + hyprlock setup that works

### DISCUSSION THREAD

**Discussion by se7en-x230** [prasanthrangan/hyprdots#1296] [General]
Any configs examples especially the exec-once line would be helpful

**Comment by kRHYME7** [comment]
https://github.com/prasanthrangan/hyprdots/pull/1173

**Comment by se7en-x230** [comment]
I finally got the hyprlock with mpris working BUT only if i starte hyprlock from the termial.
For whatever reason 
bind = $mainMod, L, exec, hyprlock # lock screen
does start hyprlock but not update the song BUT if I start it manually it does ???

![photo_6113825404453238861_w (1)](https://github.com/prasanthrangan/hyprdots/assets/69777713/121ad768-e29d-466f-8517-f7b287f2b671)

EDIT:
Same happens if hypridle starts hyprlock
My hpyridle.conf
`general {
    lock_cmd = pidof hyprlock || hyprlock 
}`

`listener {
    timeout = 60
    on-timeout = notify-send "1 Min to lock" 
}`

`listener {
    timeout = 120
    on-timeout = pidof hyprlock || hyprlock
}`
